google/oauth2/service_account.py

```diff
@@ -156,6 +159,9 @@ def __init__(
                 the JWT assertion used in the authorization grant.
             always_use_jwt_access (Optional[bool]): Whether self signed JWT should
                 be always used.
+            universe_domain (Optional[str]): The universe domain. The default
```

Comment:
Should this just be a str? What happens if `universe_domain` is `None`?

Reply:
Changed to str. Updated the logic: if `universe_domain` is `None` or "", it will be set to the default one.